### PR TITLE
CMR-7990: Added support for latency enums.

### PR DESCRIPTION
**Pull request by sxu123**
Added LOW_LATENCY and EXPEDITED enums in CollectionDataType in UMM-C v1.16.7.  
Added the same enums for dif10 and echo10 schema. The other formats don't have enums.
Added dif10 and echo10 test files.

### CODE REVIEW

File: umm-spec-lib/resources/json-schemas/collection/umm/v1.16.7/umm-search-results-json-schema.json [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+      },
+      "required": ["meta"]
+    },
+    "MetaType": {
+      "type": "object",
+      "additionalProperties": false,
+      "description": "CMR level metadata about the item found. This represents data not found in the actual metadata.",
+      "properties": {
+        "granule-count": {
+          "description": "A number >= 0 that indicates the number of granules for the collection.",
```
Comment (jaybarra) [inline umm-spec-lib/resources/json-schemas/collection/umm/v1.16.7/umm-search-results-json-schema.json:24]: I haven't noticed in others, but do we want to specify an integer number?

**Review by jaybarra — APPROVED** [review]

File: umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd [hunk 2]
```diff
@@ -260,7 +260,27 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
...
+                <xs:documentation> Data from the source that are
+                available for use within a time that is short(4-24 hours) in
+                comparison to important time scales in the phenomena
+                being studied. This data is not science quality and
+                is not retained by EOSDIS once the SCIENCE_QUALITY
+                product is archived </xs:documentation>
+              </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="EXPEDITED">
+              <xs:annotation>
```
Comment (eereiter) [inline umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd:281]: On line 246 -the documentation string for CollectionDataType, can you please change the description to the following: 
This element is used to identify the collection's ready for end user consumption latency from when the data was acquired by an instrument. NEAR_REAL_TIME is defined to be ready for end user consumption 1 to 3 hours after data acquisition. LOW_LATENCY is defined to be ready for consumption 3 to 24 hours after data acquisition. EXPEDITED is defined to be 1 to 4 days after data acquisition. SCIENCE_QUALITY is defined to mean that a collection has been fully and completely processed which usually takes between 2 to 3 weeks after data acquisition. OTHER is defined for collection where the latency is between EXPEDITED and SCIENCE_QUALITY.

On line 255 the documentation string for SCIENCE_QUALITY can you replace the text with the following:
All data products that have been fully and completely processed which usually takes between 2 to 3 weeks after data acquisition.

On line 262 the documentation string for NEAR_REAL_TIME can you replace the text with the following: 
The data is ready for end user consumption 1 to 3 hours after data acquisition. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.

On the new line 272 the documentation for LOW_LATENCY can you replace the text with the following:
The data is ready for end user consumption 3 to 24 hours after data acquisition. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.

File: umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd [hunk 3]
```diff
@@ -260,7 +260,27 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
...
+                <xs:documentation> Data from the source that are
```
Comment (eereiter) [inline umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd:282]: On the new line 282 the documentation for EXPEDITED can you replace the text with the following:
The data is ready for end user consumption 1 to 4 days after data acquisition. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.

File: umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd [hunk 4]
```diff
@@ -271,7 +291,7 @@ xmlns:xs="http://www.w3.org/2001/XMLSchema">
               <xs:annotation>
                 <xs:documentation> Any EOS and non-EOS data and data
                 products, that are not SCIENCE_QUALITY and do not
-                fall under NEAR_REAL_TIME holdings.</xs:documentation>
+                fall under NEAR_REAL_TIME, LOW_LATENCY and EXPEDITED holdings.</xs:documentation>
```
Comment (eereiter) [inline umm-spec-lib/resources/xml-schemas/echo10/Collection.xsd:294]: On line 292 the documentation for OTHER can you replace the text with the following: 
The data is ready for end user consumption between the EXPEDITED and SCIENCE_QUALITY acquisition timeframes. This data is not fully processed and is not retained by EOSDIS once the SCIENCE_QUALITY product is archived.

**Review by eereiter — CHANGES_REQUESTED** [review]

File: umm-spec-lib/resources/json-schemas/collection/umm/v1.16.7/umm-search-results-json-schema.json (hunk 1 above)
Comment (sxu123) [reply umm-spec-lib/resources/json-schemas/collection/umm/v1.16.7/umm-search-results-json-schema.json:24]: I think it's implied. I haven't changed anything in this file. It's just copied over.

**Review by eereiter — APPROVED** [review]

**Pull request merged**